### PR TITLE
fix: change poetry download url to new script

### LINKED ISSUE #8477: Docker build failing due to packages not being installed
Docker build is failing with error: `Unable to find installation candidates`
This error is showing up for a number of packages.

Possible reference issues: https://github.com/python-poetry/poetry/issues/3306   https://github.com/python-poetry/poetry/issues/3456

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #8477 

https://python-poetry.org/docs/master/#installing-with-the-official-installer

### CODE REVIEW

**Review by pc-beast — COMMENTED** [review]
```
/bin/sh: curl: not found
The command '/bin/sh -c set -eo pipefail; curl -sSL https://install.python-poetry.org | python3 -' returned a non-zero code: 127
```

File: Dockerfile [hunk 1]
```diff
@@ -13,10 +13,10 @@ RUN apk add --virtual gdk-pixbuf-dev
 ENV POETRY_HOME=/opt/poetry \
     POETRY_VIRTUALENVS_IN_PROJECT=true \
     POETRY_NO_INTERACTION=1
-    
+
 ENV PATH="$POETRY_HOME/bin:$PATH"
 
-RUN set -eo pipefail; wget -O - https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+RUN set -eo pipefail; curl -sSL https://install.python-poetry.org | python3 -
```
Comment (pc-beast) [inline Dockerfile:19]: lets use wget and ```python```

Comment (progmatic-99) [reply Dockerfile:19]: Why??

Comment (pc-beast) [reply Dockerfile:19]: curl is not there

Comment (progmatic-99) [reply Dockerfile:19]: didn't look up

**Review by pc-beast — APPROVED** [review]

**Pull request merged**